### PR TITLE
MariaDB Migration

### PR DESCRIPTION
**Pull request by hfxsd**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs/pull/15492
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: migrate-from-mariadb.md [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+
+选择合适的迁移策略：
+
+- 第一种策略是[使用 Dumpling 导出数据然后使用 TiDB Lightning 恢复](#使用-dumpling-导出数据后使用-tidb-lightning-导入)。该策略适用于所有版本的 MariaDB。该策略的缺点是需要更多的停机时间。
+- 第二种策略是[使用 DM 迁移数据](#使用-dm-迁移数据)从 MariaDB 到 TiDB。注意 DM 不支持所有版本的 MariaDB。支持的版本请参考 [DM 兼容性目录](/dm/dm-compatibility-catalog.md#tidb-data-migration-兼容性目录)。
+
+除了以上两种策略，还有其他策略适用于特定的场景，例如：
+
+- 使用 Object Relational Mapping (ORM) 工具重新部署和迁移数据。
+- 修改应用程序，使其在迁移过程中同时读写 MariaDB 和 TiDB。
```
Comment (lance6716) [inline migrate-from-mariadb.md:20]: 这里“同时读写”应该“同时写”，也就是双写就够了

**Review by lance6716 — APPROVED** [review]

File: migrate-from-mariadb.md (hunk 1 above)
Comment (hfxsd) [inline migrate-from-mariadb.md:20]:
```suggestion
- 修改应用程序，使其在迁移过程中同时写 MariaDB 和 TiDB。
```

Comment (hfxsd) [reply migrate-from-mariadb.md:20]: OK udpated

File: migrate-from-mariadb.md [hunk 2]
```diff
@@ -0,0 +1,345 @@
...
+
+# 从 MariaDB 文件迁移数据到 TiDB
+
+本文档介绍了如何将数据从 MariaDB 服务器迁移到 TiDB 集群。
+
+## 前提条件
+
+选择合适的迁移策略：
+
+- 第一种策略是[使用 Dumpling 导出数据然后使用 TiDB Lightning 恢复](#使用-dumpling-导出数据后使用-tidb-lightning-导入)。该策略适用于所有版本的 MariaDB。该策略的缺点是需要更多的停机时间。
```
Comment (ran-huang) [inline migrate-from-mariadb.md:14]:
```suggestion
- 第一种策略是[使用 Dumpling 导出数据然后使用 TiDB Lightning 恢复](#使用-dumpling-导出数据后使用-tidb-lightning-导入)。该策略适用于所有版本的 MariaDB，但缺点是需要更长的停机时间。
```

File: migrate-from-mariadb.md [hunk 3]
```diff
@@ -0,0 +1,345 @@
...
+- 第二种策略是[使用 DM 迁移数据](#使用-dm-迁移数据)从 MariaDB 到 TiDB。注意 DM 不支持所有版本的 MariaDB。支持的版本请参考 [DM 兼容性目录](/dm/dm-compatibility-catalog.md#tidb-data-migration-兼容性目录)。
+
+除了以上两种策略，还有其他策略适用于特定的场景，例如：
+
+- 使用 Object Relational Mapping (ORM) 工具重新部署和迁移数据。
+- 修改应用程序，使其在迁移过程中同时写 MariaDB 和 TiDB。
```
Comment (ran-huang) [inline migrate-from-mariadb.md:20]:
```suggestion
- 修改应用程序，使其在迁移过程中同时写入 MariaDB 和 TiDB。
```

File: migrate-from-mariadb.md [hunk 4]
```diff
@@ -0,0 +1,345 @@
...
+
+```sql
+MariaDB [test]> ALTER TABLE t DROP SYSTEM VERSIONING;
+Query OK, 0 rows affected (0.071 sec)
+Records: 0  Duplicates: 0  Warnings: 0
+```
+
+### 序列
+
+MariaDB 和 TiDB 均支持 [`CREATE SEQUENCE`](/sql-statements/sql-statement-create-sequence.md)，但是 DM 暂不支持。建议在迁移期间不要创建、修改或删除序列，尤其在迁移后不要进行相关测试。
```
Comment (ran-huang) [inline migrate-from-mariadb.md:97]: 最后一句的原文意思是“在迁移后特别测试此功能”还是“不要测试” 😂

File: migrate-from-mariadb.md [hunk 5]
```diff
@@ -0,0 +1,345 @@
...
+| TABLE_SCHEMA | TABLE_NAME |
++--------------+------------+
+| test         | s1         |
++--------------+------------+
+1 row in set (0.016 sec)
+```
+
+### 存储引擎
+
+MariaDB 为本地数据提供了存储引擎，例如 `InnoDB`、`MyISAM` 和 `Aria`。虽然 TiDB 不直接支持这些数据格式，但是你仍可以迁移这些数据。但是，也有一些存储引擎将数据放在服务器之外，例如 `CONNECT` 存储引擎和 `Spider`。虽然你可以将这些表格迁移到 TiDB，但是 TiDB 没有将数据存储在 TiDB 集群之外的功能。
```
Comment (ran-huang) [inline migrate-from-mariadb.md:122]:
```suggestion
MariaDB 为本地数据提供了存储引擎，例如 `InnoDB`、`MyISAM` 和 `Aria`。虽然 TiDB 不直接支持这些数据格式，但是你仍可以迁移这些数据。但是，也有一些存储引擎将数据放在服务器之外，例如 `CONNECT` 存储引擎和 `Spider`。虽然你可以将这些表迁移到 TiDB，但是 TiDB 无法将数据存储在 TiDB 集群外部。
```

File: migrate-from-mariadb.md (hunk 4 above)
Comment (hfxsd) [reply migrate-from-mariadb.md:97]: 应该是测试，我改下

Comment (hfxsd) [inline migrate-from-mariadb.md:97]:
```suggestion
MariaDB 和 TiDB 均支持 [`CREATE SEQUENCE`](/sql-statements/sql-statement-create-sequence.md)，但是 DM 暂不支持。建议在迁移期间不要创建、修改或删除序列，尤其在迁移后要进行相关测试。
```

File: migrate-from-mariadb.md [hunk 6]
```diff
@@ -0,0 +1,345 @@
...
+| mysql_native_password |       11 |
++-----------------------+----------+
+1 row in set (0.002 sec)
+```
+
+### 系统版本表
+
+TiDB 不支持[系统版本表 (System-Versioned Table)](https://mariadb.com/kb/en/system-versioned-tables/)。但是 TiDB 支持 [`AS OF TIMESTAMP`](/as-of-timestamp.md)，可以在某些场景下取代系统版本表。
+
+你可以执行下列语句检查受影响的表格：
```
Comment (hfxsd) [inline migrate-from-mariadb.md:66]:
```suggestion
你可以执行下列语句检查受影响的表：
```

**Review by ran-huang — APPROVED** [review]

**Pull request merged**